third_party/xla_client/stablehlo_helper.h

```diff
@@ -0,0 +1,12 @@
+#ifndef STABLEHLO_HELPER_H_
+#define STABLEHLO_HELPER_H_
```

Comment:
Curious, do you know why we need to define this macro `STABLEHLO_HELPER_H_ ` and how we use it?

Reply:
Similar to `#pragma once`, please see https://en.wikipedia.org/wiki/Include_guard